docs/basics/part-8b-using-earthly-ci.md

```diff
@@ -0,0 +1,71 @@
+The code of this example is available in the [earthly/earthly-example-ci repository on GitHub](https://github.com/earthly/earthly-example-ci).
+
+An alternative to using a traditional CI system is to use Earthly CI. With Earthly CI, you don't need any other build specification other than the Earthfile. Earthly CI uses the same technology underneath as Earthly Satellites, which means that it is dramatically faster than traditional CI systems through the use of its instantly-available cache.
+
+Earthly CI is a [paid feature](https://earthly.dev/pricing), although there is a 14-day free trial available. This section will walk you through the steps to get started with Earthly CI.
+
+## Getting started with Earthly CI
+
+To get started with Earthly CI, you need to first create an Earthly Cloud account, if you haven't already. You can do so by running the following command:
+
+```bash
+earthly account register --email <your-email>
+```
+
+Then, follow instructions in the email received. Then, create an organization, if you haven't already.
```

Comment:
Same comment about "then". Maybe change this to "Follow instructions in the email you receive. Then, create an organization if you haven't already."